java/src/main/native/src/ColumnVectorJni.cpp

```diff
@@ -573,8 +573,14 @@ JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnVector_getDeviceMemoryStringSi
     if (dtype == GDF_STRING) {
       NVStrings *nvstr = static_cast<NVStrings *>(column->data);
       return static_cast<jlong>(nvstr->memsize());
+    } else if (dtype == GDF_STRING_CATEGORY) {
+      NVCategory *cats = static_cast<NVCategory *>(column->dtype_info.category);
+      unsigned long dict_size = cats->keys_size();
+      unsigned long dict_size_total = dict_size * GDF_INT32;
+      unsigned long category_size_total = dict_size * 240;  // assumption:240B per row (10 chars in each string)
```

Comment:
It isn't clear to me how 10 characters becomes 240 bytes.  Normally 10 characters would be 10 bytes, so where are the other 230 bytes coming from?  A comment here explaining how 10 chars == 240 bytes would help.

Reply:
I think we could approximate it to 24 bytes. It's 8B aligned and considering overhead for custrings_view(8B), approximating it to 24 bytes per row.